lib/puppet/module_tool/metadata.rb

```diff
@@ -97,7 +97,7 @@ def to_hash
     end
     alias :to_data_hash :to_hash
 
-    def to_json
+    def to_json(*_args)
```

Comment:
Revisit this

Reply:
I removed this commit, it's definitely a bug, but not sure how to resolve it right now.